test/services/user/unit.test.js

```diff
@@ -178,12 +178,12 @@ describe('publicTeachers service', () => {
 	describe('TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION', () => {
 		// save TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION value
 		// eslint-disable-next-line max-len
-		const ORIGINAL_TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION = app.Config.get('TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION');
+		const ORIGINAL_TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION = Configuration.get('TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 142. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4991925)